vuepress/docs/v7.0/docs/consume/keycloak-cr.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebarDepth: 2
+---
+# EntandoKeycloakServer Custom Resource Definition 
+ 
+The EntandoKeycloakServer CRD instructs the Entando Operator to deploy the Keycloak Community OIDC provider, or its equivalent commercial product, Red Hat SSO. Once a Keycloak server has been deployed, subsequent EntandoApp and EntandoPlugin deployments can use it as an OIDC provider. The Entand Operator will automatically create OIDC Clients for these deployments in the Keycloak Server. If you already have a Keycloak instance that you want to use, skip this custom resource and use the `keycloak-admin-secret` as specified in the [External Keycloak tutorial](../../tutorials/devops/external-id-management.md).
```

Comment:
Entando Operator (missing last "o")

Keycloak server (lowercase "s" in second instance of)